docs/repositories-configure/integrations/github-integration.md

```diff
@@ -26,7 +26,7 @@ If you remove the integration, you can enable it again as follows:
 
 ## Configuring the GitHub integration {: id="configuring"}
 
-To configure the GitHub integration, open your repository **Settings**, tab **Integrations**.
+To change the configuration of the GitHub integration, open your repository **Settings**, tab **Integrations**.
```

Comment:
This change is very subtle and I'd even say that the two versions have equivalent meaning (since "to configure" can be understood as being the same thing as "to change the configuration").

I think I understand the rationale behind this edit though, but personally I'd wait until the feature is more developed before we made a more significant change to the introductions of these sections.

Reply:
The rationale for this change is that now the user is changing the default settings that are defined at the repository level. As said before on this page, "**When you add a new repository, the integration is already enabled by default with the settings defined for your organization."**

Previously, although the integration was enabled with some default values, it was like the user needed to configure the integration for the first time. 

I can hold this change if you consider it important, but I don't really under why... 🤔 

You said that this is a very subtle change... Are you suggesting that we do a more significant change later? Will we come back here if the feature doesn't get new developments?

Reply:
Just like before, when the Git provider integration is created it's already configured with some default settings (before, those defaults were built-in and couldn't be changed and now they'll be defined at the organization level).

I think that if we want to convey that users are tweaking the default configurations, we need to be more explicit. For example:

> - To customize the GitHub integration settings of a repository, [... ]
> - To adjust the GitHub integration settings of a repository, [...]

If the organization-level Git provider integration settings become more like the coding standards, we'll probably need to clarify better the implications of customizing the settings of individual repositories, and that's why I'm thinking that we'll need to refactor these instructions at some point.

Reply:
Ok, I get it. 👍

Reply:
Reverted this change.